code/modules/mining/boulder_processing/_boulder_processing.dm

```diff
@@ -350,8 +352,8 @@
 				rejected_mats[possible_mat] = quantity
 				continue
 			points_held = round(points_held + (quantity * possible_mat.points_per_unit * MINING_POINT_MACHINE_MULTIPLIER)) // put point total here into machine
-			if(!silo_materials.mat_container.insert_amount_mat(quantity, possible_mat))
-				rejected_mats[possible_mat] = quantity
```

Comment:
rejcted_mats is still used over here

https://github.com/tgstation/tgstation/blob/cade0e86107c7361df2b0abbbd32b914f18137e2/code/modules/mining/boulder_processing/_boulder_processing.dm#L359

Remove the var properly from all places